tests/foreman/ui/test_syncplan.py

```diff
@@ -119,7 +90,6 @@ def test_positive_create_with_start_time(self):
             )
             self.assertIsNotNone(self.syncplan.search(plan_name))
             self.syncplan.search(plan_name).click()
-            self.syncplan.wait_for_ajax()
```

Comment:
ok I didnt know that.  In the past I had gotten review comments for UI code to remove the wait_for_ajax(). I can put it back.